DXRMapFixups/DeusEx/Classes/DXRFixupM15.uc

```diff
@@ -79,7 +79,6 @@ function PreFirstEntryMapFixes_Bunker()
 
     // doors_lower is for backtracking
     AddSwitch( vect(4309.076660, -1230.640503, -7522.298340), rot(0, 16384, 0), 'doors_lower');
-    player().DeleteAllGoals();
 
```

Comment:
I think we had some annoyances with goals sometimes getting stuck, and Area 51 can really add a lot of goals, so maybe try this
```suggestion
    if(dxr.flags.settings.starting_map < 150) player().DeleteAllGoals();
```